crowbar_framework/app/controllers/installer/upgrades_controller.rb

```diff
@@ -156,6 +156,9 @@ def backup
     def nodes
       @current_step = 8
       if request.post?
+        @service_object = CrowbarService.new(Rails.logger)
```

Comment:
Actually I think you only need to adapt `before_filter :set_service_object` at the beginning